dev/com.ibm.ws.microprofile.health_fat/fat/src/com/ibm/ws/microprofile/health/fat/NoHealthCheckAPIImplTest.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.health.fat;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.net.HttpURLConnection;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.*;
+
+/**
+ * Example Shrinkwrap FAT project:
+ * <li> Application packaging is done in the @BeforeClass, instead of ant scripting.
+ * <li> Injects servers via @Server annotation. Annotation value corresponds to the
+ * server directory name in 'publish/servers/%annotation_value%' where ports get
+ * assigned to the LibertyServer instance when the 'testports.properties' does not
+ * get used.
+ * <li> Specifies an @RunWith(FATRunner.class) annotation. Traditionally this has been
+ * added to bytecode automatically by ant.
+ * <li> Uses the @TestServlet annotation to define test servlets. Notice that no @Test
+ * methods are defined in this class. All of the @Test methods are defined on the test
+ * servlet referenced by the annotation, and will be run whenever this test class runs.
+ */
+@RunWith(FATRunner.class)
+public class NoHealthCheckAPIImplTest {
```

Comment:
LITE or FULL? Idk if this matters because I believe default is FULL if that is what you want

Reply:
Good to consider.  Not sure the defaults.  With 12, quick tests.. it's probably for now OK to run them all in LITE mode.